packages/runtime/src/controls/rich-text/dto-types.ts

```diff
@@ -25,7 +26,8 @@ interface ValueJSON {
   annotations?: { [key: string]: AnnotationJSON } | undefined
   data?: { [key: string]: any } | undefined
 }
-export type RichTextDTO = ValueJSON
+
+export type RichTextDTO = IndexSignatureHack<ValueJSON>
```

Comment:
Wouldn't it be better to use this utility type in the places where `RichTextDTO` gets imported from this file? That way the "hack" is placed in the location where it's needed vs in this general types file, which it's not clear why it's needed here

Reply:
That's true. Moved that to a place where we need it.